util/app-config/src/tests/cli.rs

```diff
@@ -58,9 +58,6 @@ fn ba_message_requires_ba_arg_or_ba_code_hash() {
 
     let err = err.err().unwrap();
     assert_eq!(clap::error::ErrorKind::MissingRequiredArgument, err.kind());
-    assert!(err
```

Comment:
https://docs.rs/clap/4.1.8/src/clap/error/kind.rs.html#330-331

Error type assertions are stable, but error to string seems to change from version to version, including rustc errors, we should not assert specific errors